src/main/java/uk/gov/companieshouse/api/accounts/service/impl/TnepValidationServiceImpl.java

```diff
@@ -55,19 +55,10 @@ public boolean validate(String ixbrl, String location) {
             Results results = validatIxbrlAgainstTnep(ixbrl, location);
 
             if (hasPassedTnepValidation(results)) {
-                addToLog(false, null, results, location,
-                    "Ixbrl is valid. It has passed the TNEP validation");
-
                 isIxbrlValid = true;
-
-            } else {
-                addToLog(true, null, results, location,
-                    "Ixbrl is invalid. It has failed the TNEP validation");
             }
 
         } catch (Exception e) {
-            addToLog(true, e, null, location,
-                "Exception has been thrown when calling TNEP validator. Unable to validate Ixbrl");
         }
 
         LOGGER.info("TnepValidationServiceImpl: Ixbrl validation has finished");
```

Comment:
I'd suggest updating this log line to include whether the validation was successful.

Reply:
I rolled back and removed only "results" from the **addToLog**